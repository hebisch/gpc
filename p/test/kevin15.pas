program readchar(input, output, inputfile);

var
        inputfile : file of char;
        letter : char;
        i : integer;

begin
        reset(inputfile, '');
        seekread(inputfile, 3);
        writeln(inputfile^);
        for i := 1 to 5 do
        begin
          read(inputfile,letter);
          writeln(letter);
        end;
        close(inputfile);
end.
