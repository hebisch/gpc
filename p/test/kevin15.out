D
D
E
F
G
H
